core/tests/webdriverio_utils/waitFor.js

```diff
@@ -191,6 +191,19 @@ var urlRedirection = async function(url) {
     });
 };
 
+var numberOfElementsToBe = async function(
+    elementSelector, elementName, number) {
+  await browser.waitUntil(async function() {
+    var element = await $$(elementSelector);
+    return element.length === number;
+  },
+  {
+    timeout: DEFAULT_WAIT_TIME_MSECS,
+    timeoutMsg: `Number of ${elementName} is greater than ${number}\n` +
```

Comment:
This message doesn't seem to be a general case, right? What if it's less than the number if we use it somewhere else?

Reply:
Done.